tests/hook_main.py

```diff
@@ -2609,6 +2610,72 @@ def faq_get_list(transaction):
         db.session.commit()
 
 
+# ------------------------- FAQ Types -------------------------
+@hooks.before("FAQ Types > FAQ Type Collection > Create FAQ Type")
+def faq_type_post(transaction):
+    """
+    POST /faqs
+    :param transaction:
+    :return:
+    """
+    with stash['app'].app_context():
+        event = EventFactoryBasic()
+        db.session.add(event)
+        db.session.commit()
+
+
+@hooks.before("FAQ Types > FAQ Type Detail > FAQ Type Detail")
+def faq_type_get_detail(transaction):
+    """
+    GET /faqs/1
+    :param transaction:
+    :return:
+    """
+    with stash['app'].app_context():
+        faq_type = FaqTypeFactory()
+        db.session.add(faq_type)
+        db.session.commit()
+
+
+@hooks.before("FAQ Types > FAQ Type Detail > Update FAQ Type")
+def faq_type_patch(transaction):
+    """
+    PATCH /faqs/1
+    :param transaction:
+    :return:
+    """
+    with stash['app'].app_context():
+        faq_type = FaqTypeFactory()
+        db.session.add(faq_type)
+        db.session.commit()
+
+
+@hooks.before("FAQ Types > FAQ Type Detail > Delete FAQ Type")
+def faq_type_delete(transaction):
+    """
+    DELETE /faqs/1
+    :param transaction:
+    :return:
+    """
+    with stash['app'].app_context():
+        faq_type = FaqTypeFactory()
+        db.session.add(faq_type)
+        db.session.commit()
+
+
+@hooks.before("FAQ Types > Event FAQ Type Collection > List All FAQ Types for an Event")
+def faq_type_get_list(transaction):
+    """
+    GET /events/1/faqs
+    :param transaction:
+    :return:
+    """
+    with stash['app'].app_context():
+        faq_type = FaqTypeFactory()
+        db.session.add(faq_type)
+        db.session.commit()
+
+
```

Comment:
Shouldn't there also be a FAQ FAQ Type collection? Not sure about this.

Reply:
Yeah, forgot that, added now :) . Resolved.